msal/changelog.txt

```diff
@@ -1,3 +1,14 @@
+2.0.1-preview
+=============
+This release contains:
+- Resolved logger issue in B2C pm sample for msal with object null reference.
+- Adding required claims to error messages.
+- Resolved issue where B2C access and refresh tokens were not cached due to missing claims in the id Token.
+- Resolved issue with AcquireTokenSilent without passing authority when migrating from ADAL V4 to MSAL V2.
+- Resolved issue with duplicate key when caching.
+- MSAL will now use the common authority if account with null id is passed to silent api
+- Fixed issue with accounts not being removed from cache.
```

Comment:
I would write (to be customer focused):
This release contains bug fixes on top of MSAL 2.0.0-preview
- When using MSAL 2.0.0-preview with Azure AD B2C, the cache was never hit. (See MSAL#[604](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/604)), and the accounts were not removed correctly (See MSAL #[613](613))
- The `TokenCacheExtensions.Deserialize` was throwing if a null array of bytes was passed as arguments instead of silently not doing anything. See MSAL #[603](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/603) 
- When migrating a token cache from ADAL v3 or ADAL v4 to MSAL 2.0, the override of acquire token silent without authority used to work incorrectly (cache was missed)